contrib/testing/option.go

```diff
@@ -0,0 +1,63 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package testing
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext/ci"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+var (
+	// tags contains information detected from CI/CD environment variables.
+	tags map[string]string
+)
+
+type config struct {
+	skip       int
+	spanOpts   []ddtrace.StartSpanOption
+	finishOpts []ddtrace.FinishOption
+}
+
+// Option represents an option that can be passed to NewServeMux or WrapHandler.
+type Option func(*config)
+
+func defaults(cfg *config) {
+	// When StartSpanWithFinish is called directly from test function.
+	cfg.skip = 1
+	cfg.spanOpts = []ddtrace.StartSpanOption{
+		tracer.SpanType(ext.SpanTypeTest),
+		tracer.Tag(ext.SpanKind, spanKind),
+	}
+
+	// Load CI tags
+	if tags == nil {
+		tags = ci.Tags()
+	}
+
+	for k, v := range tags {
+		cfg.spanOpts = append(cfg.spanOpts, tracer.Tag(k, v))
+	}
```

Comment:
The alternative would be to include them in tracer global tags, but then it would be send for child spans too, which seems unnecessary. WDYT?